src/main/default/lwc/voteCount/__test__/voteCount.test.js

```diff
@@ -0,0 +1,122 @@
+import { createElement } from 'lwc';
+import VoteCount from 'c/voteCount';
+
+function createTestComponent() {
+  const element = createElement('c-vote-count', {
+    is: VoteCount
+  });
+  document.body.appendChild(element);
+
+  return element;
+}
+
+// Helper function to wait until the microtask queue is empty.
+function flushPromises() {
+  // eslint-disable-next-line no-undef
+  return new Promise((resolve) => setImmediate(resolve));
+}
+
+jest.mock(
+  '@salesforce/label/c.cookbook_HelpedUsers_plural',
+  () => ({ default: 'Helped 102 users' }),
+  {
+    virtual: true
+  }
+);
+
+jest.mock(
+  '@salesforce/label/c.cookbook_HelpedUsers',
+  () => ({ default: 'Helped 1 user' }),
+  {
+    virtual: true
+  }
+);
+
+jest.mock(
+  '@salesforce/label/c.cookbook_HelpedUsers_zero',
+  () => ({ default: 'Helped no one' }),
+  {
+    virtual: true
+  }
+);
+
+describe('c-vote-count', () => {
+  afterEach(() => {
+    // The jsdom instance is shared across test cases in a single file so reset the DOM
+    while (document.body.firstChild) {
+      document.body.removeChild(document.body.firstChild);
+    }
+  });
+
+  it('should display the correct count of votes when there is multiples votes', async () => {
+    const element = createTestComponent();
+    const count = 102;
+    const expectedText = `Helped ${count} users`;
+    element.count = count;
```

Comment:
Out of curiosity, how do you plan on retrieving the vote count from the platform?

Reply:
From Salesforce until proven otherwise, we plan on creating a custom object that will track probably a permanentId + a positive vote count.
And maybe a negative vote count for internal reporting?

Reply:
Ok makes sense. Thanks